man/man4/zfs.4

```diff
@@ -1154,6 +1154,11 @@ Selecting any option other than
 results in vector instructions
 from the respective CPU instruction set being used.
 .
+.It Sy zfs_blake3_impl Ns = Ns Sy 0 Ns | Ns 1 Pq int
```

Comment:
copy-pasted, fwiw

Reply:
Huh, I'm not sure how I overlooked that. Thank you.